_rules/links-with-identical-names-serve-equivalent-purpose-b20e66.md

```diff
@@ -133,7 +133,7 @@ Links created via scripting with explicit role of link, but lead to the same res
 
 A set of two SVG `<a>` elements have the same accessible name and link to the same resource.
 
-```html
+```svg
```

Comment:
Are you sure it has to be svg? 

`<svg>` creates an svg namespace, but isn't it normally embedded in an html document? 

I don't really know, and I don't know what to google for this, but it just seems a bit counter intuitive to me.